client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -75,7 +75,23 @@
 }
 
 .editor-simple-payments-modal__list {
-	padding: 8px 16px;
 	flex: auto;
 	overflow-y: auto;
 }
+
+.editor-simple-payments-modal__list-item {
+	display: flex;
+	align-items: center;
+}
+
+input[type="radio"].editor-simple-payments-modal__list-radio {
```

Comment:
The only other way I know is using `!important` :) And if I grep for `input[type=` in Calypso sources, the same trick is used at many other places to override the default styling with a class.

I'd be very interested in learning if there is some more elegant way.